plugin/execute_command.py

```diff
@@ -27,7 +27,7 @@ def run_async() -> None:
                     listener.do_signature_help_async(manual=False)
 
             return sublime.set_timeout_async(run_async)
-        session = self.session_by_name(session_name) if session_name else self.best_session(self.capability)
+        session = self.session_by_name(session_name if session_name else self.session_name)
```

Comment:
Should we also have a check later if the session has a capability to execute the command?
```
session.has_capability('executeCommandProvider')
```

Reply:
Let's do that when someone complains about it.